src/ClientHandle.cpp

```diff
@@ -3343,9 +3349,24 @@ void cClientHandle::ProcessProtocolInOut(void)
 		cCSLock Lock(m_CSIncomingData);
 		std::swap(IncomingData, m_IncomingData);
 	}
+
 	if (!IncomingData.empty())
 	{
-		m_Protocol->DataReceived(IncomingData.data(), IncomingData.size());
+		// Modifiable string view to support the protocol recogniser
+		// lopping off the initial handshake bytes:
+		std::string_view Data(IncomingData);
```

Comment:
What does this comment mean? `string_view` is by definition non-modifiable, right?

Reply:
I am passing Data in by reference and assigning to it Data.substr().

Reply:
The comment should say that then.